midealocal/devices/c3/__init__.py

```diff
@@ -309,6 +318,29 @@ def set_target_temperature(
             message.mode = mode
         self.build_send(message)
 
+    def set_customize(self, customize: str) -> None:
```

Comment:
Can we place the `set_customize` in a common class and use for all devices ?

Reply:
I don't like this. Every device type has its own logic for customize. If we'll refact that, it should go in a different PR.

Reply:
I think we are making a bit of spaghetti code, we have som many common places copied&pasted all over.

I'll approve this one, but we should really clean up the code before adding more caos.